lms/templates/courseware/course_about.html

```diff
@@ -31,49 +36,54 @@
     % if can_add_course_to_cart:
       add_course_complete_handler = function(jqXHR, textStatus) {
         if (jqXHR.status == 200) {
-          location.href = "${cart_link}";
+          location.href = "${cart_link| n, js_escaped_string}";
```

Comment:
Nit: Note that all of these should have a space before the `|`.  It would be great if you could search for and fix all of those before the next pass.  Thanks.